### DISCUSSION TITLE
JSON metadata did not contain a "client_secret" string

### DISCUSSION THREAD

**Discussion by OlivierBOEL** [OpenIDC/mod_auth_openidc#1259] [Q&A]
Hello,

I've configured dynamic registration and it works fine; unfortunately, my web server logs thousands of such warnings because OIDC provider ([https://loginuat.ntrs.com/oauth2/austt2ltc0NdZ2J8K0h7/.well-known/oauth-authorization-server](https://loginuat.ntrs.com/oauth2/austt2ltc0NdZ2J8K0h7/.well-known/oauth-authorization-server)) does not contain a "client_secret" string.
Is there a simple way to prevent this (other than changing LogLevel)?

Thanks in advance,

Olivier

**Comment by zandbelt** [comment]
not really, but your authorization server could start issuing client secrets in the registration responses since it will make web clients more secure and it won't hurt implicit clients
